01-import_and_maxfilter.py

```diff
@@ -208,6 +208,22 @@ def load_data(bids_basename):
                         extra_params=extra_params,
                         kind=config.get_kind())
 
+    # Select only the channel types we wish to analyze.
+    if config.get_kind() == 'meg' and ('mag' in config.ch_types or
+                                       'grad' in config.ch_types):
+        picks = raw.copy().pick_types(eog=True, ecg=True).ch_names
+
+        if 'mag' in config.ch_types:
+            picks += raw.copy().pick_types(meg='mag').ch_names
+        if 'grad' in config.ch_types:
+            picks += raw.copy().pick_types(meg='grad').ch_names
+    elif config.get_kind() == 'meg':
+        picks = raw.copy().pick_types(meg=True, eog=True, ecg=True).ch_names
+    else:
+        picks = raw.copy().pick_types(eeg=True, eog=True, ecg=True).ch_names
```

Comment:
It seems an overkill to copy the raw data to get ch names. Use pick functions and not the instances methods

Reply:
At this point, the data isn't preloaded though… So not _too_ much overhead! But ok, will switch to `mne.pick_types`